### DISCUSSION TITLE
libre not found building v3.4.0

### DISCUSSION THREAD

**Discussion by tbswork1** [baresip/baresip#2704] [General]
I clone libre project under baresip root, build libre and then build/install baresip, and get the following error:

```
/usr/bin/ld: ../libbaresip.a(stream.c.o): in function `rtp_handler':
stream.c:(.text+0x11fb): undefined reference to `jbuf_packets'
/usr/bin/ld: libbaresip.a(stream.c.o): in function `rtp_handler':
stream.c:(.text+0x11fb): undefined reference to `jbuf_packets'

```

**Comment by sreimers** [comment]
Looks like you build a outdated libre version, keep in mind you have always checkout/download the corresponding libre version.
If you use baresip v3.4.0 you have to use libre v3.4.0.

**Comment by tbswork1** [comment]
Because I have cloned the latest libre repo it is the latest version.
